packages/commonwealth/test/e2e/proposals.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { test, expect } from '@playwright/test';
+import { PORT } from '../../server/config';
+
+test.describe('Community proposals page', () => {
+  test.beforeEach(async ({ page }) => {
+    test.setTimeout(90000);
+  });
+
+  // Using same tests for both gov modules, so extracted first:
+  const headerTest = async ({ page }) => {
+    const activeHeader = await page
+      .locator('.Text.h3.semiBold')
+      .filter({ hasText: 'Active' })
+      .first();
+    const innerText = await activeHeader.innerText();
+    expect(innerText).toEqual('Active');
+  };
+
+  const inactiveProposalCardsTest = async ({ page }, expectedMinimumCount) => {
+    const inactiveCardsContainer = await page
+      .locator('.CardsCollection .cards')
+      .nth(1);
+    await page.waitForSelector('.ProposalCard', {
+      timeout: 60000,
+      strict: false,
+    });
+    const cardCount = await inactiveCardsContainer
+      .locator('.ProposalCard')
+      .count();
+    await expect(cardCount).toBeGreaterThanOrEqual(expectedMinimumCount);
+  };
+
+  const inactiveProposalPageTest = async ({ page }) => {
+    const inactiveCardsContainer = await page
+      .locator('.CardsCollection .cards')
+      .nth(1);
+
+    const firstCard = await inactiveCardsContainer
+      .locator('.ProposalCard')
+      .first();
+
+    const title = await firstCard
+      .locator('.Text.b1.semiBold.noWrap')
+      .first()
+      .innerText();
+
+    const navigationPromise = page.waitForNavigation();
+    firstCard.click();
+
+    await navigationPromise;
+    await page.waitForSelector('.ContentPage', {
+      timeout: 60000,
+    });
+    const content = await page.locator('.main-body-container');
+
+    const headerText = await content.locator('.header .h3').innerText();
+    const statusText = await content
+      .locator('.onchain-status-text')
+      .innerText();
+    const description = await content
+      .locator('.MarkdownFormattedText')
+      .innerHTML();
+    const voteResult = await content.locator('.VotingResult').innerHTML();
+
+    expect(headerText).toEqual(title);
+    expect(statusText).toBeDefined();
+    expect(description).toBeDefined();
+    expect(voteResult).toBeDefined();
+  };
+
+  // now the test runs:
+  test.describe('kyve-network (gov v1)', () => {
+    test.beforeEach(async ({ page }) => {
+      await page.goto(`http://localhost:${PORT}/kyve-network/proposals`);
+    });
```

Comment:
@ianrowan Any insight on why these new tests fail in CI? I'm trying to hit kyve-network/proposals and osmosis/proposals. They pass for me locally.

Maybe this DB doesn't have those records? Which db is this running against? 

Here's the test run:
https://github.com/hicommonwealth/commonwealth/actions/runs/4571405674/jobs/8069668331?pr=3247

notable: `POST /cosmosAPI/osmosis 500 299.316 ms - 49`

Reply:
@mhagel this maybe a question for @timolegros or @jnaviask. I'm not very familiar with the e2e testing setup and cosmos tests

Reply:
Moving these tests to e2eStateful, they can run in CI when @kurtisassad gets the db set up for stateful tests.